.travis.yml

```diff
@@ -19,7 +19,8 @@ services:
 before_install:
   - docker pull tangocs/mysql:9.2.2
   - docker pull tangocs/tango-cs:latest
-  - git clone https://${CI_USER_TOKEN}@github.com/tango-controls/tango-idl.git idl
+  - git clone https://github.com/JoakimSoderberg/coveralls-cmake.git
+  - git clone -b cppTango-378 https://${CI_USER_TOKEN}@github.com/tango-controls/tango-idl.git idl
```

Comment:
Right! There is no change in cppTango-378 branch in tango-idl. So this is irrelevant and does not affect the code base